presto-orc/src/main/java/com/facebook/presto/orc/writer/SliceDictionaryColumnWriter.java

```diff
@@ -226,7 +226,7 @@ public void writeBlock(Block block)
             valueCount++;
 
             if (!block.isNull(position)) {
-                // todo this is only needed if not in dictionary
+                // todo statistics only need to be updated if value was not already in the dictionary
                 statisticsBuilder.addValue(type.getSlice(block, position));
```

Comment:
Make it more explicit in the comment: min/max don't need to be updated. But nonNullCount need to be updated.